packages/core/data-transfer/lib/providers/local-strapi-destination-provider/index.ts

```diff
@@ -95,4 +97,61 @@ class LocalStrapiDestinationProvider implements IDestinationProvider {
 
     return mapSchemasValues(schemas);
   }
+
+  getAssetsStream(): NodeJS.WritableStream {
+    if (!this.strapi) {
+      throw new Error('Not able to stream Assets. Strapi instance not found');
+    }
+    const assetsDirectory = path.join(this.strapi.dirs.static.public, 'uploads');
+    return new Writable({
+      objectMode: true,
+      write(chunk, _encoding, callback) {
+        const entryPath = path.join(assetsDirectory, chunk.file);
+        const writableStream = fse.createWriteStream(entryPath);
+        chunk.stream
+          .pipe(writableStream)
+          .on('close', () => callback())
+          .on('error', callback);
+      },
+    });
+  }
+
+  getEntitiesStream(): Duplex {
```

Comment:
I believe this has been removed by another PR, you can remote it too, it will be replaced by a new version :slightly_smiling_face: